README.md

```diff
@@ -11,6 +11,10 @@ poetry update
 
 ###### Install pre-commit hooks
 
+```
+sudo apt install pre-commit
```

Comment:
That's odd here. We have pre-commit requirement in the pyproject.toml
https://github.com/Ocean-Db/fast-API/blob/main/pyproject.toml#L25
Please remove.